src/components/PhoneInput.tsx

```diff
@@ -20,17 +22,39 @@ const PhoneInput = ({ disabled, onSubmit }: { disabled?: boolean; onSubmit?: ()
 		setFullPhone(validatedPhone.phoneNumber)
 	}, [countryCode, phoneNumber])
 
+	useEffect(() => {
+		const Input = inputRef.current
+
+		if (!Input) {
+			return
+		}
+
+		const handleListMaxHeight = () => {
+			const InputBounds = Input.getBoundingClientRect()
+			setListMaxHeight(window.innerHeight - InputBounds.top - InputBounds.height)
+		}
+
+		handleListMaxHeight()
+
+		window.addEventListener('resize', handleListMaxHeight)
+
+		return () => window.removeEventListener('resize', handleListMaxHeight)
+	}, [])
+
 	return (
-		<div>
+		<Fragment>
 			<label htmlFor="phone-number" className="sr-only block text-sm font-medium text-gray-700">
 				Phone Number
 			</label>
-			<div className="relative mt-1 rounded-md shadow-sm bg-gray-100 py-4 px-3">
-				<div className="absolute inset-y-0 left-3 flex items-center">
+			<div
+				className="relative mt-1 rounded-2xl shadow-sm flex bg-gray-100 dark:bg-29343f py-4 px-3 border border-transparent focus-within:border-[#5b52f3] transition-colors"
+				ref={inputRef}
+			>
+				<div className="flex items-center">
 					<label htmlFor="country" className="sr-only">
 						Country
 					</label>
-					<CountryCodeSelect value={countryCode} onChange={setCountryCode} />
+					<CountryCodeSelect value={countryCode} onChange={setCountryCode} listMaxHeight={listMaxHeight} />
```

Comment:
@denis-ottofeller Seems like something with the country selector height.

<details>
<summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/89008845/206487168-5d39c824-adee-4966-88e7-aa5d5d442d29.png)
</details>